projects/Mallard/src/services/guardian-apple-auth-service.ts

```diff
@@ -0,0 +1,41 @@
+interface IdentityPostBody {
+    authorizationCode: string
+    idToken: string
+    givenName: string
+    familyName: string
+}
+
+function guardianAppleAuthService<T>(
+    authorizationCode: string,
+    idToken: string,
+    givenName: string,
+    familyName: string,
+) {
+    const API_URL = 'https://id.guardianapis.com/auth'
```

Comment:
you might be able to import ID_API_URL from constants.ts here

Reply:
:+1: